ui-react/packages/atlasmap/src/Views/ColumnMapperView/Columns/commonActions.tsx

```diff
@@ -88,6 +88,7 @@ export function commonActions({
           aria-label="Show mapping details"
           tabIndex={0}
           isDisabled={connectedMappings.length === 0}
+          data-testid={"show-mapping-details-button"}
```

Comment:
There will be multiple instance of all the buttons in this file, is that ok?

Reply:
These `commonActions` buttons should be ok, because they only appear on hover, so left it as it was. Other data-testid's should be ok now.